### DISCUSSION TITLE
Demo glitch: Selecting a continent updates countries but doesn't show selected country code

### DISCUSSION THREAD

**Discussion by joewiz** [Jinntec/Fore#200] [Q&A] [answered]
In https://jinntec.github.io/Fore/demo/selects.html, selecting a continent (e.g., Asia) automatically selects the first country in the resulting list (e.g., Afghanistan), but the `fx:output` that should show the country code doesn't do so. Only when the user selects a *different* country is the country code shown. To show Afghanistan's country code, then, the user has to select, say, Bahrain and then select Afghanistan. 

Is this control working as designed?

**Comment by JoernT** [comment, marked as answer]
this is a bit confusing i confess. Adding a 

`selection="open"` to the second select fixes it - the second control will stay empty after selecting the continent until user select a country.

Guess this was due to the fact that the `selection` attribute was implemented later that the demo. Have changed it.
